### DISCUSSION TITLE
Question About GitHub's Terms of Service

### DISCUSSION THREAD

**Discussion by jesterb0206** [community/community#38898] [New to GitHub] [answered]
Hello everyone that's reading this,

I hope your day has gone well so far and you haven't encountered too many pesky bugs! I have a quick question that I was wondering if somebody knew the answer to. Is it within GitHub's Acceptable Use Policies to add a link to a GoFundMe fundraiser in your profile README? I wanted to spread the word about a fundraiser a friend of mine organized but I won't add the link to my profile README if it breaks the terms and conditions set forth by GitHub.

Bradley

**Comment by jfullstackdev** [comment, marked as answer]
even GitHub has its own GitHub Sponsors

as per documentation, a pure business side / commercial is what is not allowed

**Reply by jesterb0206** [reply]
Thank you so much for your response, I appreciate it! I hope you have a good rest of your day/evening/etc.

**Reply by jfullstackdev** [reply]
there is also the tricky part, for example, you have the full business side and you will be providing link for that site using GitHub, you might be flagged because it is in sense, using GitHub to promote full commercial / business

as long as you are just providing projects and just asking for those who want to support you, then that's fine whether its through GitHub Sponsors or an external one
